docs/user-guide/mvd-using.md

```diff
@@ -74,35 +92,41 @@ Use the Languages setting in the Preferences panel to change the desktop languag
 
 ### IFrame Sample
-The IFrame sample application plug-in for developers demonstrates how to embed pre-made webpages within the desktop as an application and how an application can request an action of another application (see the source code for more information).
+Github Sample Repo:
+[sample-iframe-app](https://github.com/zowe/sample-iframe-app)
 
-### z/OS Subsystems
-The z/OS Subsystems plug-in helps you find information about the important services on the mainframe, such as CICS, Db2, and IMS.
+### Sample Angular App
+Github Sample Repo:
+[sample-angular-app](https://github.com/zowe/sample-angular-app/blob/lab/step-1-hello-world/README.md)
+
+### Sample React App
+Github Sample Repo:
+[sample-react-app](https://github.com/zowe/sample-react-app/blob/lab/step-1-hello-world/README.md)
 
 ### 3270 Terminal
-The 3270 Terminal plug-in provides a user interface that emulates the basic functions of IBM 3270 family terminals. On the "back end," the plug-in and the Zowe Application Server connect to any standard TN3270E server.
+The 3270 Terminal plugin provides a user interface that emulates the basic functions of IBM 3270 family terminals. On the "back end," the plugin and the Zowe Application Server connect to any standard TN3270E server.
 
 ### VT Terminal 
-The VT Terminal plug-in provides a user interface that emulates the basic functions of DEC VT family terminals. On the "back end," the plug-in and the Zowe Application Server connect to VT compatible hosts, such as z/OS UNIX System Services (USS), using standard network protocols.
+The VT Terminal plugin provides a user interface that emulates the basic functions of DEC VT family terminals. On the "back end," the plugin and the Zowe Application Server connect to VT compatible hosts, such as z/OS UNIX System Services (USS), using standard network protocols.
 
 ### API Catalog
-The API Catalog plug-in lets you view API services that have been discovered by the API Mediation Layer. For more information about the API Mediation Layer, Discovery Service, and API Catalog, see [API Mediation Layer Overview](../getting-started/overview.md).
+The API Catalog plugin lets you view API services that have been discovered by the API Mediation Layer. For more information about the API Mediation Layer, Discovery Service, and API Catalog, see [API Mediation Layer Overview](../getting-started/overview.md).
 
 ### Editor
-With the Zowe Editor you can create and edit files and view datasets on the system that Zowe serves.
-
-### Workflows
-From the Workflows application plug-in you can create, manage, and use z/OSMF workflows to manage your system.
+With the Zowe Editor you can create, edit, and modify the properties of files such as ownership and permissions. You can also view and edit datasets. Navigation is controlled by the File Tree https://github.com/zowe/zlux-file-explorer. Other actions are available using the top left menu, a toolbar above the tree, or hotkeys.
```

Comment:
`[File Tree](https://github.com/zowe/zlux-file-explorer)`

You could change it to something like that for a cleaner hyperlink